envconsul/plan.sh

```diff
@@ -0,0 +1,25 @@
+pkg_name=envconsul
+pkg_origin=core
+pkg_version=0.7.3
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("MPL-2.0")
+pkg_description="Launch a subprocess with environment variables using data from @HashiCorp Consul and Vault."
+pkg_upstream_url=https://github.com/hashicorp/envconsul
+pkg_source="https://releases.hashicorp.com/${pkg_name}/${pkg_version}/${pkg_name}_${pkg_version}_linux_amd64.zip"
+pkg_shasum="67ed44cb254da24ca5156ada6d04e3cfeba248ca0c50a5ddd42282cbafde80bc"
+pkg_filename="${pkg_name}-${pkg_version}_linux_amd64.zip"
+pkg_build_deps=(core/unzip)
```

Comment:
Might be best to add `pkg_deps=()` to be explicit about the runtime requirements.

Reply:
I think we decided if travis doesn't complain, then we can live without `pkg_deps=()`.